api_intro.py

```diff
@@ -1,24 +1,31 @@
 from datetime import date
+import json
 
 import requests
 
-import config
+# Get configuration data from config.json
+try:
+    with open("config.json", "r") as config_file:
```

Comment:
These should be single quotes.
In addition, `config.json` should be added to `.gitignore` file in case someone pushes their config accidentally.

Reply:
This has been updated. And the .gitignore has been added.